### PR TITLE
Huginn deployable Camera now come in packs of Three as intended.

### PR DESCRIPTION
**Pull request by Derrick2Electricboogaloo**
## About The Pull Request

This fixes an oversight in req regarding the deployable cameras, it says 3 on the list, but when purchased only one comes in a crate. This fixes that issue.

## Why It's Good For The Game

Fixes are good and now functions as described. This has been a known issue for sometime but largely ignored since it is not a very important, but here we are.

## Changelog
:cl:
fix: Deployable Camera in req comes in pack of 3 as intended.
/:cl:

### CODE REVIEW

**Comment by Derrick2Electricboogaloo** [comment]
Ah heck what went wrong, so simple >:(

**Comment by Derrick2Electricboogaloo** [comment]
Worked fine on my test server, hmmmm

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -83,7 +83,11 @@ OPERATIONS
 
 /datum/supply_packs/operations/deployablecams
 	name = "3 Deployable Cameras"
-	contains = list(/obj/item/deployable_camera)
+	contains = list(
+		/obj/item/deployable_camera,
+		/obj/item/deployable_camera,
+		/obj/item/deployable_camera,
+		)
```
Comment (Walarks) [inline code/modules/reqs/supplypacks.dm:90]: 1 less tab :^)

**Review by Walarks — CHANGES_REQUESTED** [review]

File: code/modules/reqs/supplypacks.dm (hunk 1 above)
Comment (Derrick2Electricboogaloo) [reply code/modules/reqs/supplypacks.dm:90]: Fricken tabs! Ahh

Comment (Derrick2Electricboogaloo) [reply code/modules/reqs/supplypacks.dm:90]: Yeah I thought it was the deployablecams needing to be deployable_camera, but no change on my end, your makes more sense.

**Comment by Derrick2Electricboogaloo** [comment]
LINTERS HELL YEAH!

**Review by BraveMole — APPROVED** [review]

**Pull request merged**